google/datalab/bigquery/_view.py

```diff
@@ -186,7 +186,7 @@ def results(self, use_cache=True, dialect=None, billing_tier=None):
     """
     output_options = QueryOutput.table(use_cache=use_cache)
     return self._materialization.execute(output_options, dialect=dialect,
-                                               billing_tier=billing_tier).results
+                                               billing_tier=billing_tier).result()
```

Comment:
I did miss _query.py and _bigquery.py.  Fixed those and did a sweep but didn't find any other places.